site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,96 @@
+---
+
+## Philosophy and approach
+
+Dojo 2 is grounded in the belief that accessibility is as important online as it is in our built environments and architects of both share a similar responsibility to provide access to all. Instances of poor accessibility abound in each setting and are very obvious once one starts looking:
+
+| ![Bad wheelchair ramp example](https://smhigley.github.io/assets/ramp-bad.jpg)  | &nbsp; | ![Good ramp example](https://smhigley.github.io/assets/ramp-good.jpg) |
+|:---:|:---:|:---:|
+| Afterthought Accessibility | &nbsp; | Designed Accessibility |
+
+Web developers have additional challenges since sites with poor accessibility are not obvious, except to those users that are poorly served. Without proper testing, it is very easy to add a `title` attribute to a link and a `for` attribute to a `<label>` and think accessibility requirements have been met.
+
+The guidelines provided by the Web Accessibility Initiative ([WAI-ARIA](https://www.w3.org/WAI/intro/aria)) are very good, but they can lead to the false impression that accessibility (a11y) is achieved by simply adding an `aria-labelledby` or `role` attribute onto a finished widget. The only approach that can guarantee equal access is to think about accessibility during the design phase and continue through development and testing. For that reason, Dojo 2 integrates a11y into its widget system at a fundamental level; all of its widgets have been designed to be accessible by default, and any tools needed to meet the Web Content Accessibility Guidelines ([WCAG](https://www.w3.org/WAI/intro/wcag)) have been integrated throughout Dojo 2's widget system.
+
+## Using Dojo 2 widgets
+
+All widgets provided through the `@dojo/widgets` package have been created with a range of assistive technology and perceptual differences in mind. However, they are not foolproof. There are a number of instances where additional information is required before peak accessibility can be achieved. Often, a few properties are required to enable a widget is to be fully accessible by all users. To help developers achieve this goal, each widget’s example page models best practices. Additionally, many of the properties exposed by a widget are related to accessibility, providing valid values for those properties allows a developer to be sure that they are not missing any a11y enhancements.
+
+### Form controls
+
+All simple form controls, such as `TextInput`, `Textarea`, and `Select`, use either a styled native control or provide a native fallback, which allows them to take advantage of built-in accessibility features wherever possible. Properties are provided in the widget interface to control relevant ARIA (Accessible Rich Internet Applications Suite) attributes like `aria-invalid`, `aria-describedby`, and `aria-readonly`.
+
+For example, a `TextInput` widget with the following properties:
```

Comment:
To help with the following sentence, may want to reword this one slightly, e.g. "For example, consider a `TextInput` widget with the following properties:"